.github/workflows/benchmark.yml

```diff
@@ -44,27 +44,19 @@ jobs:
           cd benchmarks
           export BASE=origin/${{ github.base_ref }}
           export HEAD=origin/${{ github.head_ref }}
-          # run asv benchmarks on the target
-          # -s 1 ensures only one "step" (the tip of the branch) is run
-          asv run $BASE -s 1
-          # run asv on the head of this PR
-          asv run $HEAD -s 1
+          asv continuous $BASE $HEAD
+
+      - name: Write stats to archive on failure
+        if: ${{ failure() }}
```

Comment:
I'm not worried either way, but I'm still curious: why only store and upload the results on a failure?

Reply:
I'm happy to make this an always if there is utility in access to the data 👍

Reply:
This is now `always()` so results will be uploaded on both success and failure